src/setup/rgw_setup

```diff
@@ -116,11 +142,12 @@ class ResetCmd(SetupCmdBase):
     def __init__(self, args):
         """ Initialize super class members."""
         super().__init__(args)
-        self.config_path = args.config
+        self._conf = self._load_config(args.config)
```

Comment:
Why dont we move this line to the base class since its called from all the derived classes.

Reply:
done.